.travis.yml

```diff
@@ -1,3 +1,6 @@
+env:
+- API_REGEX='Google\.(Cloud\.)?[A-M].*'
```

Comment:
Will this skip Google.LongRunning?

Reply:
Nope, I thought of that - that's why Cloud is an optional segment :)